packages/dx-react-scheduler-material-ui/src/templates/confirmation-dialog/layout.jsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+export const Layout = React.memo(({
+  buttonComponent: Button,
+  handleClose,
+  confirm,
+  getMessage,
+  isDeleting,
+  appointmentData,
+  ...restProps
+}) => (
+  <div
+    {...restProps}
+  >
+    <DialogTitle>{getMessage(isDeleting ? 'confirmDeleteMessage' : 'confirmCancelMessage')}</DialogTitle>
+    <DialogActions>
+      <Button onClick={handleClose} title={getMessage('cancelButton')} />
+      <Button
+        onClick={confirm}
```

Comment:
Can we make these actions similar? I mean `handleCancel` and for example `handleAccept`.